.travis.yml

```diff
@@ -15,22 +16,17 @@ cache:
 
 env:
     global:
+        - APP_ENV=test
         - SYLIUS_CACHE_DIR=$HOME/.sylius-cache
         - SYLIUS_BUILD_DIR=etc/build
-    matrix:
-        - SYMFONY_VERSION="3.4.*"
-        - SYMFONY_VERSION="4.1.*"
 
 before_install:
     - phpenv config-rm xdebug.ini
     - echo "memory_limit=4096M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
     - mkdir -p "${SYLIUS_CACHE_DIR}"
-
-    - cp tests/Application/.env.test.dist tests/Application/.env.test
-    - set -a && source tests/Application/.env.test && set +a
+    - mkdir -p tests/Application/web/media/image
```

Comment:
Let's add `tests/Application/web/media/image/.gitignore` file instead of creating it in Travis CI, so that running tests manually for the first time won't fail.